### DISCUSSION TITLE
Paid Support Plans

### DISCUSSION THREAD

**Discussion by StefanKoell** [AvaloniaUI/Avalonia#8598] [General]
I'm considering switching to AvaloniaUI (from WinForms/WPF) for a commercial app and I'm also interested in some sort of paid support plan. 300 USD/mo/seat is kind of steep for a small commercial app like mine. If I look at 3rd party vendors who offer UI suites (like DevExpress), they usually charge around 500 - 1000 USD/year. I'm wondering if there are plans to offer plain technical support plans at a cheaper rate, e.g. email support only or something like that for us smaller shops?

**Comment by danwalmsley** [comment]
@StefanKoell as much as we would love to be able to offer some cheaper solution, even on the current plans it costs us money to run the support.

Most support customers > 10 hours of requests per week. Even small numbers of seats. 

Im sure you understand how much it costs to hire developers to do this work. 

We can offer work and support or training on an hourly rate basis, if you need something like this please get in touch via email... (details on the website).

Otherwise we have an active and engaged community on telegram and they some level of support is available there for free.

https://t.me/Avalonia

**Comment by StefanKoell** [comment]
Thanks for the quick response. I totally understand. It's impressive what you've achieved and I'm definitely checking out AvaloniaUI. I hope it will not come to this but having support available on an hourly rate is a good alternative and gives a bit of peace of mind.
